adapters/httpapi_compact.c

```diff
@@ -56,6 +56,7 @@ typedef struct HTTP_HANDLE_DATA_TAG
     unsigned int    is_io_error : 1;
     unsigned int    is_connected : 1;
     unsigned int    send_completed : 1;
+    unsigned int    tls_renegotiation;
```

Comment:
Shall be `bool`. Please see comment below.

Reply:
as a bit flag...right?

Reply:
Fixed on the commit 98666e9b4d46e63d900d4abd1bc1591d3049b5a6